spec/classes/rules/qemu_spec.rb

```diff
@@ -77,7 +77,6 @@
         end
 
         it { is_expected.to compile }
-        it { is_expected.to have_resource_count(0) }
```

Comment:
If I'm not wrong you're removing this assert because `nftables` is present now when the class is tested. However, removing it reduces test coverage as the point of this assert was to make sure that `nftables::rules::qemu` didn't add any resource if all the toggles were off. I'd suggest to then actively assert that neither of the rules created by `nftables::rules::qemu` are present to have the same test coverage as before the patch.